includes/ampli-vs-amplitude-link-to-core-sdk.md

```diff
@@ -0,0 +1,4 @@
+???info "Ampli Wrapper versus the Amplitude SDK"
```

Comment:
This seems superfluous with the info included in the overview:

> The Ampli Wrapper is a generated, strongly typed API for tracking Analytics events based on your Tracking Plan in Amplitude Data. The tracking library exposes a function for every event in your team’s tracking plan. The function’s arguments correspond to the event’s properties.
> 
> Ampli can benefit your app by providing autocompletion for events & properties defined in Data and enforce your event schemas in code to prevent bad instrumentation. 
> 
> Ampli supports Node.js apps written in JavaScript (ES6 and higher) and TypeScript (2.1 and higher). The generated tracking library is packaged as a CJS module.

Reply:
Thanks, I agree, I had the info block mostly to provide a link back to the untyped Amplitude SDK. I will try to reduce the redundancy or move the link.